lib/charms/tempo_k8s/v0/tracing.py

```diff
@@ -86,7 +86,9 @@ def __init__(self, *args):
 DEFAULT_RELATION_NAME = "tracing"
 RELATION_INTERFACE_NAME = "tracing"
 
-IngesterProtocol = Literal["otlp_grpc", "otlp_http", "zipkin", "tempo"]
+IngesterProtocol = Literal[
+    "otlp_grpc", "otlp_http", "zipkin", "tempo", "jaeger_thrift", "jaeger_grpc"
```

Comment:
I'd say uniformly rename jaeger_thrift to jaeger_http_thrift as that seems to be the official protocol name